.travis.yml

```diff
@@ -5,14 +5,14 @@ node_js:
   - "11"
   - "12"
 addons:
-  firefox: "latest"
+  chrome: stable
   apt:
     sources:
     - ubuntu-toolchain-r-test
     packages:
     - g++-4.8
 before_install:
-  - sh -e /etc/init.d/xvfb start
+  - google-chrome-stable --headless --disable-gpu --remote-debugging-port=9222 http://localhost &
```

Comment:
Is this necessary? I have a project using Karma and Chrome headless, and I've never seen this before.

Reply:
I've directly taken this from the Travis docs, might be outdated and reference an older Linux version. Wait, will directly try without and re-report here on the results.

Reply:
Ah, you're right, not needed.

Ready for a re-review.